lib/github_changelog_generator/parser_file.rb

```diff
@@ -41,12 +43,18 @@ def open_settings_file
     end
 
     def parse_line!(line, line_number)
+      return if non_configuration_line?(line)
       option_name, value = extract_pair(line)
       @options[option_key_for(option_name)] = convert_value(value, option_name)
     rescue
       raise ParserError, "Failed on line ##{line_number}: \"#{line.gsub(/[\n\r]+/, '')}\""
     end
 
+    # Returns true if the line starts with a pound sign or a semi-colon.
+    def non_configuration_line?(line)
+      line =~ /^[\#;]/ || line =~ /^[\s]+$/
```

Comment:
All good. Just questiona  about ";"-comment syntax?
Why strings, starts from `;` should specified as comment? 
First time, when see such syntax.